src/main/wlp/server.xml

```diff
@@ -1,8 +1,7 @@
 <server description="new server">
 
     <!-- Enable features -->
-<featureManager>
-       <feature>jms-2.0</feature>

```

Comment:
We determined that wasJmsServer and wasJmsClient both pull in the jms-2.0 feature when they are installed, so it doesn't need to be set as a feature.